dist/app/app.server.js

```diff
@@ -0,0 +1,35 @@
+"use strict";
+var __createBinding = (this && this.__createBinding) || (Object.create ? (function(o, m, k, k2) {
+    if (k2 === undefined) k2 = k;
+    Object.defineProperty(o, k2, { enumerable: true, get: function() { return m[k]; } });
+}) : (function(o, m, k, k2) {
+    if (k2 === undefined) k2 = k;
+    o[k2] = m[k];
+}));
+var __setModuleDefault = (this && this.__setModuleDefault) || (Object.create ? (function(o, v) {
+    Object.defineProperty(o, "default", { enumerable: true, value: v });
+}) : function(o, v) {
+    o["default"] = v;
+});
+var __importStar = (this && this.__importStar) || function (mod) {
+    if (mod && mod.__esModule) return mod;
+    var result = {};
+    if (mod != null) for (var k in mod) if (k !== "default" && Object.prototype.hasOwnProperty.call(mod, k)) __createBinding(result, mod, k);
+    __setModuleDefault(result, mod);
+    return result;
+};
+Object.defineProperty(exports, "__esModule", { value: true });
+exports.initServer = void 0;
+const express_1 = __importStar(require("express"));
+const bread_router_1 = require("./bread/bread.router");
+const app = express_1.default();
+const router = express_1.Router();
+app.use(router);
+router.use(bread_router_1.router);
+let initServer = (port) => {
+    app.listen(port, () => {
+        console.log(`server started at http://localhost:${port}`);
+    });
+};
+exports.initServer = initServer;
+//# sourceMappingURL=app.server.js.map
```

Comment:
Delete all the dist files 😄

Reply:
Already added correctly the dist folder into gitignore